src/theme/base/_variables.scss

```diff
@@ -1,11 +1,22 @@
+// Font Family
 $font-main: 'Open Sans', sans-serif;
 $font-fjalla-one: 'Fjalla One', sans-serif;
-$font-giant: 5.25rem;
 
+// Palette
 $color-main: #EB6E64;
 $color-white: #FFF;
 $green300: #32B88C;
 $gray29: #4A4A4A;
 $gray: #E8E7E7;
 $black: #000;
 $green500: #69D5B1;
+
+// Font Sizes
+$font-xxs: 0.75rem;     // 12px
+$font-xs: 0.875rem;     // 14px
+$font-s: 1rem;          // 16px
+$font-m: 1.125rem;      // 18px
+$font-l: 1.5rem;        // 24px
+$font-xl: 2rem;         // 32px
+$font-xxl: 4.25rem;     // 68px
+$font-giant: 5.25rem;   // 84px
```

Comment:
Can we just change this to xxxl? I know that's a lot of x's but giant just doesn't fit with the naming convention :)

Reply:
Done :)

Reply:
Thanks! Approved!